docs/setup_docs.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+HERE=` basename "$PWD"`
+if [[ "$HERE" != "docs" ]]; then
+    echo "Please only execute in the docs/ directory";
+    exit 1;
+fi
+
+# Create a default go.mod file
+cat <<EOF >go.mod
+module github.com/apache/flink
+
+go 1.18
+EOF
+
+echo "Created temporary file" $goModFileLocation/go.mod
+
+# Make Hugo retrieve modules which are used for externally hosted documentation
+currentBranch=$(git branch --show-current)
```

Comment:
Please create a follow-up ticket to derive this variable from `docs/config.toml#branch`.

Reply:
https://issues.apache.org/jira/browse/FLINK-27474